.github/workflows/pr.yml

```diff
@@ -40,24 +40,32 @@ jobs:
     uses: ./.github/workflows/env-create-file.yml
     secrets: inherit
 
+  setup:
+    name: Yarn install & cache
+    uses: ./.github/workflows/yarn-install.yml
+    secrets: inherit
+
```

Comment:
With yarn v3 this is way faster. I still need to make some test if skipping this makes significant speed improvements.